model/config_test.go

```diff
@@ -150,7 +147,7 @@ func TestConfigIsValidDefaultAlgorithms(t *testing.T) {
 
 	err := c1.SamlSettings.isValid()
 	if err != nil {
```

Comment:
You no longer need the if here, is enough the `require.Nil(t, err, "SAMLSettings validation should pass with default settings")` line here.